tests/test_sftp_automatic.py

```diff
@@ -0,0 +1,123 @@
+from base import TestSFTPBase
+import tap_tester.connections as connections
+import tap_tester.menagerie   as menagerie
+import tap_tester.runner      as runner
+import os
+import csv
+import json
+
+class TestSFTPAutomatic(TestSFTPBase):
+
+    def name(self):
+        return "tap_tester_sftp_automatic_fields"
+
+    def get_files(self):
```

Comment:
It seems duplicating can we move this to base.py?

Reply:
Yes, Moved get_files() to base for the most used common structure, and override in other files if requires.